### ISSUE TITLE
VAAPI on Wayland broken on 106.0.5197.0

### ISSUE THREAD

**Issue by pkochans** [Alex313031/Thorium#57]
After update to 106.0.5197.0 hardware acceleration stopped working on Wayland, also fonts rendering looks fuzzy. Verified using intel_gpu_top.
It was working well on the previous version (unsure which it was, I'm using deb repo) so my system is configured properly.

**Comment by Alex313031** [comment]
@pkochans Strange, lets see if it still happens with next release.

**Comment by pkochans** [comment]
Nevermind, got it working again `--ozone-platform=wayland --enable-features=UseOzonePlatform --enable-features=VaapiVideoDecoder --use-gl=egl --disable-features=UseChromeOSDirectVideoDecoder`

**Comment by Alex313031** [comment]
@pkochans You don't have to use --disable-features=UseChromeOSDirectVideoDecoder or --enable-features=VaapiVideoDecoder These are already set this way in source code as part of the vaapi patches in thorium

**Comment by pkochans** [comment]
@Alex313031 I experimented a bit and either `--ozone-platform=wayland` or `--ozone-platform-hint=auto` is sufficient. Now I can see that in chrome://flags/ there is no longer any results for "ozone". Some switch was there in the previous version and it served the same as "ozone-platform-hint". I've set it to auto on the previous version but since it dissipated I have to add those flags again. So nothing broken, just labs flag got removed. Thanks!
